### ISSUE TITLE
Work Package wp-bh.csv

### ISSUE THREAD

**Issue by neilernst** [bhermann/DoR#51]

**Comment by XiaoLing941212** [comment]
I would like to work on this!

**Comment by kpeng2019** [comment]
I would like to work on this please!

**Comment by bhermann** [comment]
Hi @XiaoLing941212 . Is there an update on this work package?

**Comment by XiaoLing941212** [comment]
@bhermann yes I have 5 results for 5 packages... the tricky part is Windows cannot upload .csv file to this comment... Trying to find a possible way to do that

**Comment by XiaoLing941212** [comment]
[wp-bh-results.txt](https://github.com/bhermann/DoR/files/6792502/wp-bh-results.txt)

Results for phase 1.

**Comment by XiaoLing941212** [comment]
I completed this work package.

**Comment by bhermann** [comment]
I'll write you for a Zoom session next week, okay?

**Comment by XiaoLing941212** [comment]
okay!

**Comment by XiaoLing941212** [comment]
@bhermann updated result for this package
[wp-bh-results.csv.txt](https://github.com/bhermann/DoR/files/6826130/wp-bh-results.csv.txt)
